CONTRIBUTING.md

```diff
@@ -157,3 +159,8 @@ To get a local preview of the documentation:
 Then you can go to Docker Desktop settings and open the resources, add the temporary directory path `/tmp`.
 
 > Note that `make docs` uses a lot of memory. If it crashes, increase the memory allocated to Docker and try again.
+
+Also note that PRs are merged to the main branch.  If you want to see your changes published to the latest release, add the appropriate backport label to your PR.
```

Comment:
I think we need to be more clear if we expect the contributor to add the label themselves.

Alternatively, I think it is good practice for the maintainer to ask on these kinds of questions on the PR, either of themselves (do I want to see this fix in "latest" as soon as possible?) or of the contributor, and for the maintainer to add the label.